alchimia/engine.py

```diff
@@ -162,6 +162,9 @@ def first(self):
     def keys(self):
         return self._deferrer(self._result_proxy.keys)
 
+    def close(self):
+        return self._result_proxy.close()
```

Comment:
This needs to be like all the other methods, and use `self._deferrer`, and return a `Deferred`; `_result_proxy.close()` is a potentially blocking method so doing it this way would potentially (A) illegally share a connection object between threads and (B) block the main loop.